sdks/python/apache_beam/options/pipeline_options_validator.py

```diff
@@ -102,7 +102,7 @@ def validate(self):
     """
     errors = []
     for cls in self.OPTIONS:
-      if 'validate' in cls.__dict__:
+      if 'validate' in [k for k, v in cls.__dict__.items() if str(v).startswith("<function")]:
```

Comment:
Can you just do: `if 'validate' in cls.__dict__ and callable(cls.__dict__['validate'])`?

Also, please make sure that the line is less than 80-char long : )

Reply:
got it, i will change